packages/react/src/components/Panel/Panel.styles.ts

```diff
@@ -361,7 +360,7 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
           },
         },
       },
-      isFooterSticky && {
```

Comment:
i've a feeling that this test was there for a reason.

Reply:
it's mostly the top border color....when it's not sticky the border shouldn't be there. 

![image](https://user-images.githubusercontent.com/1434956/160934670-138c032f-f662-45b9-af3f-9af37bd63c22.png)

As this is really an edge case, can you simply add some styles to the footer for the times that you delay render content into the footer?

Reply:
could that reason be obsolete now since the footer is always sticky?

Reply:
if it's just the `borderTopColor`, could we do this? I think there are others who would appreciate the change

![image](https://user-images.githubusercontent.com/8689181/160935894-ebbac0be-6004-4f7d-b761-e620df0b5a84.png)

Reply:
nevermind, then the behavior on `borderTopColor` would not be consistent in this edge case scenario

Reply:
One other option is just force the content area to be bigger, and therefore the 'isFooterSticky' will always trigger

```
 <Panel
        styles={{
          content:{
                flexShrink: 0,
                height: '90%'
               }
               }}
```

Reply:
The issue is caused by this: https://github.com/microsoft/fluentui/pull/21466/files#:~:text=background%3A%20semanticColors.bodyBackground%2C

Seems like the `borderTopColor` had this issue before, where it would not show on a dynamic body at first until it resizes (due to `isStickyFooter` not setting after the dynamic body renders). We're only noticing it now because with the change linked above, the footer's background is also not set.